src/components/Cards/Login.vue

```diff
@@ -1,36 +1,36 @@
 <template>
-  <v-card class="elevation-12 my-16" width="500">
+  <v-card class="elevation-12 my-16" width="477px">
     <v-toolbar dark color="primary">
       <v-toolbar-title>{{title}}</v-toolbar-title>
     </v-toolbar>
     <v-card-text>
       <v-form>
         <v-text-field
-        id="inputEmail"
-        prepend-icon="email"
-        name="login"
-        label="Email address"
-        type="email"
-        required
-        v-model="email"
-        :disabled="loggingIn"
-        :error-messages="error['email']"
+          id="inputEmail"
+          prepend-icon="email"
+          name="login"
+          label="Email address"
+          type="email"
+          required
+          v-model="email"
+          :disabled="loggingIn"
+          :error-messages="error['email']"
         />
         <v-text-field
-        id="inputPassword"
-        prepend-icon="lock"
-        name="password"
-        label="Password"
-        type="password"
-        required
-        v-model="password"
-        :disabled="loggingIn"
-        :error-messages="error['password']"
+          id="inputPassword"
+          prepend-icon="lock"
+          name="password"
+          label="Password"
+          type="password"
+          required
+          v-model="password"
+          :disabled="loggingIn"
+          :error-messages="error['password']"
         />
       </v-form>
-      <router-link to="/forgotten-password">Forgotten password</router-link>
+      <router-link to="/forgotten-password" style="text-decoration: none">Forgot your password?</router-link>
```

Comment:
I wonder if we need to add the right (24px) padding here. Maybe using something like `py-6` added to the class list?

Reply:
The padding bottom of the login field is 16px and the padding top of the login button is default to 8px. So luckily I set the right spacing to it accidentally.